### DISCUSSION TITLE
Support enum values in Astro DB

### DISCUSSION THREAD

**Discussion by zellwk** [withastro/roadmap#980] [Proposal]
# Summary

Support enum values in Astro DB

# Background & Motivation

Having `enum` values can help users check the database against specific values and reduce data problems. 

Since, enum values are already present in Drizzle, supporting them should be simple. From what I'm observing so far, it seems like we just need to add an `enumValues` field. 

# Goals

- Give users better configurability over their database, supporting data integrity. 

# Example 

```js
export const User = defineTable({
  columns: {
    role: column.text({ enum: ['admin', 'mod', 'member'] })
  },
})
```

**Comment by florian-lefebvre** [comment]
Enum support was added in 5.13! https://astro.build/blog/astro-5130/#enum-support-in-astro-db-tables
